pkg/reconciler/ksvc/resources/knative_configuration_test.go

```diff
@@ -194,7 +194,7 @@ func TestKnativeConfiguration(t *testing.T) {
 								TimeoutSeconds: &constants.DefaultTimeout,
 							},
 							Container: &v1.Container{
-								Image: "animeshsingh/sklearnserver:",
+								Image: v1alpha1.SKLearnServerImageName + ":latest",
```

Comment:
Did we test it out once after the build? Don`t think we have actual end to end test deploying and running predictions against the built images....there is one which checks the route.

Reply:
I did docker pull docker run. I'll do it again after my changes.